### PR TITLE
Gok-424 | Add. to decode secured Aadhaar qr code

### PR DESCRIPTION
**Pull request by SanoferSameera**
Co-authored-by:yenugukeerthana <keerthana.yenugu@thoughtworks.com>

### CODE REVIEW

File: app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarCard.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.example.abha_create_verify_android.utils
+
+class AadhaarCard {
```
Comment (mohan-13) [inline app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarCard.kt]: Can we rename this as AadhaarCardInfo?

File: app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarSecureQr.kt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import java.io.IOException
+import java.io.UnsupportedEncodingException
+import java.math.BigInteger
+import java.util.LinkedList
+import java.util.zip.GZIPInputStream
+
+
+class AadhaarSecureQr(scanData: String?) {
+    private var decodedData: ArrayList<String>? = null
+    private var scannedAadhaarCard: AadhaarCard = AadhaarCard()
```
Comment (mohan-13) [inline app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarSecureQr.kt]: How about scannedAadharInfo ?

File: app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarCard.kt (hunk 1 above)
Comment (SanoferSameera) [reply app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarCard.kt]: done

File: app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarSecureQr.kt (hunk 2 above)
Comment (SanoferSameera) [reply app/src/main/java/com/example/abha_create_verify_android/utils/AadhaarSecureQr.kt]: done

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**